src/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -157,7 +157,9 @@ public sealed partial class ProcessStartInfo
         public ProcessStartInfo() { }
         public ProcessStartInfo(string fileName) { }
         public ProcessStartInfo(string fileName, string arguments) { }
+        public ProcessStartInfo(string fileName, System.Collections.Generic.IReadOnlyCollection<string> argumentList) { }
```

Comment:
Was this ctor approved as part of the API design review?  Sorry if I missed it.

Reply:
I wasn't at the review, but I don't think so. @Anipik you were going to propose this separately and forgot to revert this to private?

Reply:
Also note as @AtsushiKan this would be a source breaking change as the overload would be ambiguous if the 2nd parameter was null. That's the kind of thing they consider in api review.

Reply:
> you were going to propose this separately and forgot to revert this to private?

Note, though, that with the property not having a setter and its getter returning a read-only collection, it's useless, so something in the design has to change.

Reply:
@danmosemsft I was going to ask the review for Process.Start() overload. You are on a thread where I asked about adding the constructor.